externals/kyuubi-flink-sql-engine/bin/flink-sql-engine.sh

```diff
@@ -37,7 +37,7 @@ FLINK_SQL_ENGINE_HOME="$(cd `dirname $0`/..; pwd)"
 if [[ "$FLINK_SQL_ENGINE_HOME" == "$KYUUBI_HOME/externals/engines/flink" ]]; then
   FLINK_SQL_ENGINE_LIB_DIR="$FLINK_SQL_ENGINE_HOME/lib"
   FLINK_SQL_ENGINE_JAR=$(find "$FLINK_SQL_ENGINE_LIB_DIR" -regex ".*/kyuubi-flink-sql-engine_.*\.jar")
-  FLINK_HADOOP_CLASSPATH="$INTERNAL_HADOOP_CLASSPATHS"
+  FLINK_HADOOP_CLASSPATH="$INTERNAL_HADOOP_CLASSPATHS:$HADOOP_CLASSPATH"
```

Comment:
IMO, the `$INTERNAL_HADOOP_CLASSPATHS` includes the `$HADOOP_CLASSPATH`. Why do you add the `HADOOP_CLASSPATH`?

Reply:
You may refer to the #2082

Reply:
You may refer to the #2082